demo/admin/src/news/blocks/NewsListBlock.tsx

```diff
@@ -20,13 +20,12 @@ export const NewsListBlock: BlockInterface<NewsListBlockData, State, NewsListBlo
     defaultValues: () => ({ ids: [] }),
     AdminComponent: ({ state, updateState }) => {
         const { scope } = useContentScope();
-        const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("NewsListBlock") };
+        const dataGridProps = {
+            ...useDataGridRemote(),
+            ...usePersistentColumnState("NewsListBlock"),
+        };
         const intl = useIntl();
 
-        const columns: GridColDef<GQLNewsListBlockNewsFragment>[] = [
```

Comment:
Is it okay to remove the explicit type here? Does DataGrid have type inference for other props?

Reply:
what do you mean with other props? All props used in here, are in MUI DataGrids prop definition of GridColDef?

Reply:
I meant props like `valueGetter` or `renderCell`. Anything that may use fields from the provided `rows`.